spv/sync.go

```diff
@@ -79,9 +79,22 @@ type Syncer struct {
 // Notifications struct to contain all of the upcoming callbacks that will
 // be used to update the rpc streams for syncing.
 type Notifications struct {
-	// Synced is defined for a callback to notify when the wallet is seen
-	// to be synced or unsynced from its connected peers.
-	Synced func(sync bool)
+	Synced                       func()
+	Unsynced                     func()
+	FetchMissingCFiltersStart    func()
```

Comment:
To match the past tense of "Finished", we should use "Started" for all of these.